File: app/src/main/java/com/BlizzardArmory/warcraft/Equipment/Type.java

package com.BlizzardArmory.warcraft.Equipment;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Type {

    @SerializedName("type")
    @Expose
    private String type;
    @SerializedName("name")
    @Expose
    private String name;

    public String getType() {
        return type;
    }

    public void setType(String type) {
        this.type = type;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

}

File: app/src/main/java/com/BlizzardArmory/model/warcraft/equipment/Type.kt
package com.BlizzardArmory.model.warcraft.equipment

import androidx.annotation.Keep
import com.google.gson.annotations.SerializedName


/**
 * The type Type.
 */
@Keep
data class Type(

    @SerializedName("type")
    var type: String,

    @SerializedName("name")
    var name: String

)